taxi/urls.py

```diff
@@ -38,11 +42,18 @@
         name="manufacturer-delete",
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
+    path("cars/<int:car_id>/adduser/", add_user, name="car-add-user"),
+    path("cars/<int:car_id>/deleteuser/", delete_user, name="car-delete-user"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/create", DriverCreateView.as_view(), name="driver-create"),
```

Comment:
Consider adding a trailing slash to the 'drivers/create' path for consistency with other URL patterns. It should be 'drivers/create/'.